app/services/organizations/update_service.rb

```diff
@@ -55,12 +60,17 @@ def update_from_api(params:)
       if params.key?(:billing_configuration)
         billing = params[:billing_configuration]
         organization.invoice_footer = billing[:invoice_footer] if billing.key?(:invoice_footer)
-        organization.invoice_grace_period = billing[:invoice_grace_period] if billing.key?(:invoice_grace_period)
         organization.vat_rate = billing[:vat_rate] if billing.key?(:vat_rate)
+        if billing.key?(:invoice_grace_period)
+          Organizations::UpdateInvoiceGracePeriodService.call(organization:, grace_period: billing[:invoice_grace_period])
```

Comment:
Layout/LineLength: Line is too long. [122/120] (https://rubystyle.guide#max-line-length)